### PR TITLE
Update aboutMe.md and add tiongMax.md

### PR DESCRIPTION
**Pull request by tiongMax**

### CODE REVIEW

**Comment by brein62** [comment]
```
ERROR:../tp/.idea/checkstyle-idea.xml:15: no newline at EOF.
ERROR:../tp/.idea/vcs.xml:6: no newline at EOF.
ERROR:../tp/.idea/workspace.xml:4[5](https://github.com/AY2324S1-CS2103T-T14-4/tp/actions/runs/6390840227/job/17344923610?pr=38#step:5:6): no newline at EOF.
```

**Comment by brein62** [comment]
now just add the image then youre set 👍

**Review by ruo-x — APPROVED** [review]
lgtm!!

File: docs/AboutUs.md [hunk 1]
```diff
@@ -50,12 +50,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Tiong Meng Sheng
 
 <img src="images/johndoe.png" width="200px">
```
Comment (brein62) [inline docs/AboutUs.md:55]: now just add the image then youre set 👍

**Review by brein62 — CHANGES_REQUESTED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (jordankanghm) [reply docs/AboutUs.md:55]: agreed! please use a heart-stopping, blood-pumping, rizz-inducing picture of yourself!

**Review by brein62 — APPROVED** [review]
photo to be updated later!

**Pull request merged**